autogpt_platform/frontend/src/components/agptui/AgentRunStatusChip.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Badge } from "@/components/ui/badge";
+
+export type AgentRunStatus =
+  | "success"
+  | "failed"
+  | "queued"
+  | "running"
+  | "stopped"
+  | "scheduled"
+  | "draft";
```

Comment:
should this live in types or is it specific to this

Reply:
It will become part of the app-wide data model later, but for now there's this slight difference between what we want to show and what the API produces, and this is how I'm dealing with it.